cli/src/operations.py

```diff
@@ -0,0 +1,27 @@
+DEFAULT_NUM_IMAGES = 40
+
+
+class MissingConfigException(Exception):
+    pass
+
+
+def init(config):
+    if (config is None):
+        raise MissingConfigException()
+
+    raise NotImplementedError
+
+
+def download(num_images):
+    if (num_images is None):
```

Comment:
@bnookala We might want to have upper and lower limits on the number of images to download

Reply:
@andrebriggs fixed!